queries/editorconfig/highlights.scm

```diff
@@ -0,0 +1,56 @@
+(comment) @comment @spell
+
+; consistency with toml and ini
+(section
+  (section_name) @type)
```

Comment:
I disagree; this should be `@markup.heading` or `@module` ("namespaces").

Reply:
Actually, aren't these always paths or globs? Shouldn't we use `@string.special.path` or `@string.regexp` in that case (and possibly inject the `regex` parser)?